torch/quasirandom.py

```diff
@@ -50,13 +50,13 @@ def __init__(self, dimension, scramble=False, seed=None):
             if self.seed is not None:
                 g.manual_seed(self.seed)
 
-            self.shift = torch.mv(torch.randint(2, (self.dimension, self.MAXBIT), generator=g),
-                                  torch.pow(2, torch.arange(0, self.MAXBIT, dtype=torch.double))).to(torch.long)
+            self.shift = torch.mv(torch.randint(2, (self.dimension, self.MAXBIT), dtype=torch.float, generator=g),
```

Comment:
This was very confusing to me: `torch.randint` by name is supposed to return an `int` but isn't. Is this intentional or a bug?